practice/00-HTML-CSS-basics/Image video and sound.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>page</title>
+  </head>
+<body>
+	image
+	<img src="http://www.clarin.com/extrashow/Argentina-Holanda-Brasil-copa-del-mundo-Javier-Mascherano-Sergio-Romero_CLAIMA20140709_0231_27.jpg"/>
+	sound
+	<embed src="1-01 Tema de Pototo.mp3"> 
```

Comment:
For audio and video, use the html5 tags audio and video.

https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/Using_HTML5_audio_and_video